app/src/routes/lookup/+server.js

```diff
@@ -1,31 +1,21 @@
+import {PUBLIC_ONTOLOGY_API_HOST} from '$env/static/public'
 import {json} from '@sveltejs/kit'
 
+
 /** @type {import('./$types').RequestHandler} */
-export async function GET({url: {searchParams}, platform}) {
+export async function GET({url: {searchParams}, locals: {db}}) {
 	/** @type {LookupTerm} */
 	const word = searchParams.get('word') ?? ''
 
-	if (!platform?.env.DB_Ontology) {
-		console.error(`Ontology database missing from platform arg: ${JSON.stringify(platform)}`)
-
-		return response({term: word, matches: []})
-	}
-
-	let matches = await get_matches_from_ontology(platform.env.DB_Ontology)(word)
+	let matches = await get_matches_from_ontology(word)
 
 	if (matches.length > 0) return response({term: word, matches})
 
-	if (!platform?.env.DB_Editor) {
-		console.error(`Editor database missing from platform arg: ${JSON.stringify(platform)}`)
-
-		return response({term: word, matches})
-	}
-
 	/** @type {string} */
-	const stem = await check_inflections(platform.env.DB_Editor)(word)
+	const stem = await check_inflections(db)(word)
```

Comment:
check_inflections() should strip the sense before checking, then put it back on before returning. So something like `sons-C` finds the stem `son`, then searches the ontology for `son-C`

Reply:
if that wasn't doing that before this PR then let's do that in another PR... this one was just about changing the lookup from a local DB to a REST call.